label_studio/ui_test/readme.txt

```diff
@@ -0,0 +1,72 @@
+UI Automation for Label-studio
+
+What we have?
+_Automation script
+_testCases.txt file
+_files for import (jpg, png, txt, …)
+
+testCases.txt file contains all needed types(Like: image, font, common formats, audio, html, time series)
+And also it contains appropriate formats for each type (Example: jpg, png for image)
+the symbol ‘+’ means that current format will be tested
+the symbol ‘-‘ means that current format will not be tested
+
+[testCases.txt file]
+=====image=====
++jpg
++png
+-gif
+----------
+=====font=====
++txt
+----------
+=====common formats=====
++csv
+-tsv
```

Comment:
why those are not tested?

Reply:
This gives us to testing in different ways.
Sure we can do test for all file types by setting '+' before each file type,
but also we can do testing for chooses formats only.
For example
* if we want to do test only for images, then we will set '+' before image formats(jpg, png,...)
* if we want to test only one format 'jpg' we will just set '+' before jpg and '-' before another formats (this can be used for case, when developers changed something only for jpg format, and there is a need to do testing only for that format)